assets/dev/js/editor/elements/views/base.js

```diff
@@ -577,11 +587,7 @@ BaseElementView = BaseContainer.extend( {
 		}
 	},
 
-	renderOnChange( settings ) {
-		if ( ! this.allowRender ) {
-			return;
-		}
-
+	renderChanges( settings, shouldRenderUI = true ) {
```

Comment:
who is passing the arg `shouldRenderUI` and why it is required? (Maybe add explanation in comment)

Reply:
Handled.